templates/404.html

```diff
@@ -7,8 +7,17 @@
 {% block content %}
 <div class="p-strip is-deep">
   <div class="row">
-    <div class="col-4">
-      <img src="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg" alt="Error owl" width="400" height="400" class="p-image--smaller-mobile"/>
+    <div class="col-4 u-align--center">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg",
+          alt="Error owl",
```

Comment:
In this case, is not an owl.
```suggestion
          alt="Error 404",
```